docs/release-notes/13-2-0/README.md

```diff
@@ -3,22 +3,87 @@ title: OpenProject 13.2.0
 sidebar_navigation:
     title: 13.2.0
 release_version: 13.2.0
-release_date: TBD
+release_date: 2023-01-17
 ---
 
 # OpenProject 13.2.0
 
-Release date: TBD
+Release date: 2023-01-17
 
 We released [OpenProject 13.2.0](https://community.openproject.com/versions/1979).
-The release contains several bug fixes and we recommend updating to the newest version.
 
+It brings improvements for the **OneDrive/SharePoint integration**, now also allowing **Manual project folders**. In addition to that, both OneDrive/SharePoint and Nextcloud integrations were improved by **showing deleted files in OpenProject work packages**.
 
-## Important updates and breaking changes
+We also added the option to **filter the project member list**, allowing project administrators to easily filter through the project member lists based on various roles, groups and shares. Instance administrators can now also allow for users to **change work package status without the rights to edit a work package**. 
 
-<!-- Remove this section if empty, add to it in pull requests linking to tickets and provide information -->
+Furthermore, it is now **possible to set users with whom a work package has been shared as assignees or responsible**. The **quick content menu** in the Gantt view has now shows work package relations. 
```

Comment:
The **quick content menu** in the Gantt view has now shows work package relations. --> This sounds slightly off, suggestion: 
The **quick content menu** in the Gantt view now offers an additional option to show work package relations.